adapters/ix/ix.go

```diff
@@ -18,46 +19,69 @@ import (
 	"github.com/prebid/openrtb/v19/openrtb2"
 )
 
+const (
+	FtHandleMultiImpOnSingleRequest = "pbs_handle_multi_imp_on_single_req"
+)
+
+const ExpireFtTime = 3600 //1 hour
```

Comment:
Nitpick: Calling this `expireFtTimeSeconds` might help provide more context in addition to the comment. However, please see my other comments regarding the server keeping this kind of state.